cmd/passage/cmd/root.go

```diff
@@ -265,6 +272,17 @@ func (a appCreator) newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, a
 		wasmOpts = append(wasmOpts, wasmkeeper.WithVMCacheMetrics(prometheus.DefaultRegisterer))
 	}
 
+	// validate minimum-gas-prices value is greater than or equal to defaultMinGasPrice value
+	minGasPricesStr := cast.ToString(appOpts.Get(server.FlagMinGasPrices))
+	minGasPrices, err := sdk.ParseDecCoins(minGasPricesStr)
+	if err != nil {
+		panic(err)
+	}
+	if minGasPrices.AmountOf(baseDenom).LT(sdk.MustNewDecFromStr(defaultMinGasPrice)) {
+		panic(fmt.Sprintf("minimum-gas-prices value in app.toml should be greater than or equal to %s%s",
+			defaultMinGasPrice, baseDenom))
+	}
```

Comment:
Should we do this at server side (inside ante or somewhere) to handle it better for gRPC, RPC calls as well?